### PR TITLE
More server typings

### PR DESCRIPTION
**Pull request by LavrovArtem**

### CODE REVIEW

**Comment by LavrovArtem** [comment]
@testcafe-build-bot retest-all

**Comment by LavrovArtem** [comment]
@testcafe-build-bot retest

File: @types/match-url-wildcard/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+declare module 'match-url-wildcard' {
```
Comment (miherlosev) [inline @types/match-url-wildcard/index.d.ts]: Move typings to the module. If it's not possible create a separate module `@types/<module_name>` and publish it.

File: src/request-pipeline/context.ts [hunk 2]
```diff
@@ -55,7 +55,7 @@ interface ContentInfo {
     isRedirect: boolean;
 }
 
-interface OnResponseEventData {
+export interface OnResponseEventData {
```
Comment (miherlosev) [inline src/request-pipeline/context.ts]: Place in `typing` folder all exported interfaces.

**Comment by LavrovArtem** [comment]
@testcafe-build-bot retest

**Comment by LavrovArtem** [comment]
@testcafe-build-bot retest

**Comment by LavrovArtem** [comment]
@testcafe-build-bot retest

**Pull request merged**